src/js/components/Tip/Tip.js

```diff
@@ -63,7 +63,9 @@ const Tip = forwardRef(({ children, content, dropProps, plain }, tipRef) => {
         target={componentRef.current}
         trapFocus={false}
         key="tip-drop"
-        plain
+        background="none"
```

Comment:
Do we even need these? I was expecting `theme.tip.drop` to take care of resetting anything from the themed drop styling.

Reply:
That would have been true if we had default `theme.tip.drop` props that are overriding the `global.drop` styles, but we do not. I'm reluctant to add those due to backward compatibility concerns.
So if the Drop isn't `plain`, and the theme are not overriding the base values, Tip will pick up all of its default theme styling of `global.drop` as follows:
```
      drop: {
        background: {
          dark: 'black',
          light: 'white',
        },
        border: {
          radius: '0px',
        },
        shadowSize: 'small', // shadowSize is deprecated, use 'elevation'
        zIndex: '20',
      },
```
Hence, we need the explicit props suggested on this PR to override those default values.
I'm open to suggestions on cleaner ideas.

Reply:
Would it be essentially equivalent to `theme.tip.drop = { background: 'none', elevation: 'none', margin: 'none' }`?

Reply:
Yes, I added backward compatibility tests for the theme to verify it wouldn't break anything. The change had been made.